.github/workflows/dart.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - main
   pull_request:
+    paths-ignore:
```

Comment:
If there are only changes in `sentry_logging`, there's no need to build either the dart or flutter sdk because they don't depend on `sentry_logging`

Reply:
that makes sense